### ISSUE TITLE
ERROR: cartography --neo4j-uri bolt://127.0.0.1:7687

### ISSUE THREAD

**Issue by andres-ortiz** [lyft/cartography#60]
Hello
I have a error when I launch `cartography --neo4j-uri bolt://127.0.0.1:7687`.

Error is :
```
ERROR:cartography.sync:Unhandled exception during sync stage 'aws'
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/cartography/sync.py", line 63, in run
    stage_func(neo4j_session, config)
  File "/usr/lib/python3.6/site-packages/cartography/intel/aws/__init__.py", line 121, in start_aws_ingestion
    _sync_multiple_accounts(session, aws_accounts, regions, config.update_tag, common_job_parameters)
  File "/usr/lib/python3.6/site-packages/cartography/intel/aws/__init__.py", line 62, in _sync_multiple_accounts
    _sync_one_account(session, boto3_session, account_id, regions, sync_tag, common_job_parameters)
  File "/usr/lib/python3.6/site-packages/cartography/intel/aws/__init__.py", line 20, in _sync_one_account
    iam.sync_group_policies(session, boto3_session, account_id, sync_tag, common_job_parameters)
  File "/usr/lib/python3.6/site-packages/cartography/intel/aws/iam.py", line 382, in sync_group_policies
    load_group_policies(neo4j_session, groups_policies, aws_update_tag)
  File "/usr/lib/python3.6/site-packages/cartography/intel/aws/iam.py", line 278, in load_group_policies
    action = statement.get('Action')
AttributeError: 'str' object has no attribute 'get'
```

```
# python --version
Python 3.6.8
```
I run cartography in docker with neo4j:3.3.9 as image.

Thank you for your help !

**Comment by achantavy** [comment]
Hey there, it looks like that `action = statement.get('Action')` code failed at line 278 (see [this line](https://github.com/lyft/cartography/commit/4054483930e510a2211f68a1e2524007e0657078#diff-baa4279abf52541ddd5fd08a4374f512R278)), and it looks like this got fixed [recently](https://github.com/lyft/cartography/commit/e7f7fce98ae2bd8fad3ac6f9f91d4ebbc220e66c#diff-baa4279abf52541ddd5fd08a4374f512R279).

It looks like this change has not been released yet but we will push it out as soon as we can since it's crashing.

**Comment by andres-ortiz** [comment]
Thank you !

**Comment by achantavy** [comment]
Fixed in #63 , check out the new 0.4.0 release
